CHANGES.rst

```diff
@@ -1,40 +1,57 @@
-Changelog
-=========
+.. _vunreleased:
+
+unreleased
+^^^^^^^^^^
+
+- Bring repo up to Pylons Project standards.
+  See https://github.com/Pylons/pyramid_mailer/pull/89
+
+- Optionally include bcc information on send.
+  See https://github.com/Pylons/pyramid_mailer/pull/87
+
+.. _v0.15.1:
```

Comment:
Are the custom anchors standard now in Pylons project repo changelogs?  If so, then we should add them for other releases, too.  If not, then drop it.

Reply:
Fixed in 8cb79e5ab576344216d10aef01b170a593e0017e